drv/ksz8463/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 [dependencies]
 num-traits = { workspace = true }
 
+counters = {path = "../../lib/counters"}
```

Comment:
Is this change (and the one to `drv/ksz8463/src/lib.rs`) intentional?  It looks like a merge / rebase glitch.